src/book/03-guides/04-general/04-visualmesh.mdx

```diff
@@ -0,0 +1,201 @@
+---
+section: Guides
+chapter: General
+title: Visual Mesh Getting Started
+description: How to get started with the Visual Mesh.
+slug: /guides/general/visualmesh
+---
+
+This guide compliments the quickstart guide on the [main Visual Mesh repo](https://github.com/Fastcode/VisualMesh) by being geared towards those with little experience of the [Linux command line (i.e. bash)](https://ubuntu.com/tutorials/command-line-for-beginners#1-overview) or of Machine Learning workflows. If you are already familiar with bash and machine learning, you may prefer to work off the Quickstart guide on the Visual Mesh repo as it is briefer than this guide.
+
+To get started, run this line to check the installation and version of the required programs.
+
+```bash
+python3 --version && pip3 --version && git --version && docker --version
+```
+
+_this should provide a result similar to that listed below, possibly with higher version numbers, run through the installation guides if this doesn't appear_
+
+```bash
+Python 3.8.5
+pip 20.0.2 from /usr/lib/python3/dist-packages/pip (python 3.8)
+git version 2.25.1
+Docker version 20.10.3, build 48d30b5
+
+```
+
+<details>
+
+<Summary>Installing Python3</Summary>
```

Comment:
Haven't confirmed this, but the build errors you're getting are likely due to these uppercase tag names. Should be all lowercase like so:

```suggestion
<summary>Installing Python3</summary>
```

Because this is a [MDX](https://mdxjs.com/) file, tag names beginning with uppercase are treated as React [JSX components](https://reactjs.org/docs/introducing-jsx.html), not HTML tags (like [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details), [`<summary>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary), etc). This is how we implement NUbook's custom tags like [`<Alert>`](https://nubook.netlify.app/system/tools/nubook#alerts-and-warnings), [`<Grid>`](https://nubook.netlify.app/system/tools/nubook#content-in-grids), etc.

So it's looking for a `<Summary>` React component but failing because one doesn't exist. Switching to `<summary>` will use the HTML tag instead, and should work.

Reply:
Looks logical - I suspected the problem was the tags and JSX, but I didn't know enough about JSX to know what to look for beyond that.